Changelog.md

```diff
@@ -1,3 +1,16 @@
+### 0.8.9
+
+Language Features:
+
+
+Compiler Features:
+ * Commandline Interface: Disallowed the ``--experimental-via-ir`` option to be used with Standard Json, Assembler and Linker modes
```

Comment:
The next version is 0.8.8. This should be added to the existing section below.

Reply:
Added